vendors/espressif/boards/ports/ota_pal_for_aws/aws_esp_ota_ops.c

```diff
@@ -0,0 +1,507 @@
+// Copyright 2015-2016 Espressif Systems (Shanghai) PTE LTD
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <stdint.h>
```

Comment:
Did you modify this file? Seems much larger than the previous version. I would have guessed that we just needed to move this file and not change it.

Reply:
Looks like there were some additional functions added by the team during previous iterations. These are helper functions to facilitate some PAL methods.